presto-main/src/main/java/com/facebook/presto/operator/repartition/PartitionedOutputOperator.java

```diff
@@ -419,15 +444,21 @@ public void flush(boolean force)
 
                     operatorContext.recordOutput(pagePartition.getSizeInBytes(), pagePartition.getPositionCount());
 
-                    List<SerializedPage> serializedPages = splitPage(pagePartition, DEFAULT_MAX_PAGE_SIZE_IN_BYTES).stream()
-                            .map(serde::serialize)
-                            .collect(toImmutableList());
-
-                    outputBuffer.enqueue(operatorContext.getDriverContext().getLifespan(), partition, serializedPages);
+                    outputBuffer.enqueue(operatorContext.getDriverContext().getLifespan(), partition, splitAndSerializePage(pagePartition));
                     pagesAdded.incrementAndGet();
                     rowsAdded.addAndGet(pagePartition.getPositionCount());
                 }
             }
         }
+
+        private List<SerializedPage> splitAndSerializePage(Page pagePartition)
+        {
+            List<Page> split = splitPage(pagePartition, DEFAULT_MAX_PAGE_SIZE_IN_BYTES);
```

Comment:
Can we rename `split` to `splittedPages`? It's easy to get confused on the word`split` with Presto splits

Reply:
Sure, renamed.